### DISCUSSION TITLE
Check cards for Recommended at new Beta View

### DISCUSSION THREAD

**Discussion by jacovinus** [community/community#14338] [Other Feature Feedback, Questions, & Ideas]
**please check cards size at new Beta View** 
![recomended-cards](https://user-images.githubusercontent.com/30432825/161805452-f33382cf-c421-4855-bbc2-ec8c97a045ad.png)

**Comment by alex-kim-dev** [comment]
also mentioned here https://github.com/github/feedback/discussions/14301
